### PR TITLE
S2 | Complete an item

### PR DESCRIPTION
**Pull request by ardagokce**
#32

### CODE REVIEW

File: users-api/app.js [hunk 1]
```diff
@@ -7,12 +7,19 @@ const bodyParser = require('body-parser');
 //create app
 const app = express();
 
+<<<<<<< HEAD
```
Comment (raghunat) [inline users-api/app.js]: you need to fix this. This is a conflict. Choose the portions of code that reflect your changes and mine

File: hunts-api/routes/complete.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// complete.js
+
+const express = require('express');
+
+const router = express.Router();
+
+router.post('/', (req,res) => {
+    if((req.body.id == 'abcd-efgh-ijkl-mnop') && (req.body.image == 'base64String')){
```
Comment (raghunat) [inline hunts-api/routes/complete.js]: This isn't important, but you don't need the extra set of parenthesis here

File: hunts-api/routes/complete.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      res.status(200).send('SUCCESS');
```
Comment (raghunat) [inline hunts-api/routes/complete.js]: These responses should be json objects like mentioned in the acceptance criteria of the ticket:

```js
res.json({
  "status": "SUCCESS"
})
```

**Comment by raghunat** [comment]
@ardagokce this still needs work

File: hunts-api/routes/complete.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// complete.js
+
+const express = require('express');
+
+const router = express.Router();
+
+router.post('/', (req,res) => {
+    if((req.body.id == 'abcd-efgh-ijkl-mnop') && (req.body.image == 'base64String')){
+      res.status(200).send(res.json({
```
Comment (raghunat) [inline hunts-api/routes/complete.js]: @ardagokce no .send needed, just

`res.status(200).json({`

**Comment by raghunat** [comment]
@ardagokce just conflicts left to resolve

File: hunts-api/app.js [hunk 5]
```diff
@@ -5,10 +5,20 @@ const bodyParser = require('body-parser');
...
+
 app.use('/version', require('./routes/version.js'));
 
 app.use('/hunts', require('./routes/hunts.js'));
 
+app.use('/complete/item', require('./routes/complete.js'));
+
 app.listen(3000, () => console.log('App is listening...'));
+
+app.listen(process.env.PORT || 3000, () => console.log('App is listening...'));
```
Comment (raghunat) [inline hunts-api/app.js]: You didn't resolve the conflicts well, if you look, you duplicated code with the above line

**Comment by ardagokce** [comment]
________________________________
Gönderen: Stephen Raghunath <notifications@github.com>
Gönderildi: 10 Nisan 2018 Salı 00:14
Kime: raghunat/scavenger-hunt
Bilgi: ardagokce; Mention
Konu: Re: [raghunat/scavenger-hunt] S2 | Complete an item (#35)


@raghunat requested changes on this pull request.

You had some issues resolving the conflicts. Can you also run the launcher script and paste the output here?

________________________________

In hunts-api/app.js<https://github.com/raghunat/scavenger-hunt/pull/35#discussion_r180232402>:

>  app.listen(3000, () => console.log('App is listening...'));
+
+app.listen(process.env.PORT || 3000, () => console.log('App is listening...'));


You didn't resolve the conflicts well, if you look, you duplicated code with the above line

**Pull request merged**